### PR TITLE
add rank to shipment lines display

### PR DESCRIPTION
**Pull request by altairis-tof**

### CODE REVIEW

**Comment by hregis** [comment]
@altairis-tof you can't not use "position" instead "rang"?

**Comment by altairis-tof** [comment]
> @altairis-tof you can't not use "position" instead "rang"?

@hregis position is not a field of commandedet

File: htdocs/expedition/shipment.php [hunk 1]
```diff
@@ -623,6 +623,7 @@
 			$i = 0;
 			print '<thead>';
 			print '<tr class="liste_titre">';
+			print '<th>'.$langs->trans("Rank").'</th>';
```
Comment (eldy) [inline htdocs/expedition/shipment.php:626]: To follow other document, can you add the condition 
` if (getDolGlobalString('MAIN_VIEW_LINE_NUMBER')) {`

Comment (altairis-tof) [reply htdocs/expedition/shipment.php]: ok it is fixed

**Pull request merged**